english/wrestle-hand/lasso-hand.md

```diff
@@ -0,0 +1,3 @@
+You quickly dive, grabbing the rope and fork, as the marshmallow hand grasps for your leg.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.